src/AssetManager.Infrastructure/Data/AssetManagerContext.cs

```diff
@@ -14,5 +14,7 @@ public class AssetManagerContext : DbContext
         DbSet<Supplier> Suppliers { get; set; }
 
         DbSet<Depreciation> Depreciations { get; set; }
+
+        DbSet<Category> Categories { get; set; }
```

Comment:
Please add **Access Modifier**, example-

```csharp
public DbSet<Category> Categories { get; set; } 
```

Although it will work with default property **Access Modifier**, `private` but we should use `public`

Reply:
Access Modifier changed